app/templates/events/view/tickets/add-order.hbs

```diff
@@ -0,0 +1,46 @@
+<div class="row" style="padding-top:10px">
+  <h2 class="ui header">{{t 'Add Order'}}</h2>
+</div>
+<table class="ui very basic table">
+  <thead>
+    <tr>
+      <th>{{t 'Ticket Type'}}</th>
+      <th class="right aligned">{{t 'Price '}}(US$)</th>
+      <th class="right aligned">{{t 'Quantity'}}</th>
+      <th class="right aligned">{{t 'Item Total'}}</th>
+    </tr>
+    <tr>
+      <td>{{t 'Free'}}</td>
+      <td class="right aligned">{{t 'US$ 0.00'}}</td>
+      <td class="right aligned">{{input value=quantity_item1 type='number' min='0'}}</td>
+      <td class="right aligned">{{input value=total_item1 type='number' min='0'}}</td>
```

Comment:
Use `type='text'`, This represents the total of the item i.e (price * number of items).